models/game.rb

```diff
@@ -16,6 +16,7 @@ class Game < Base
       SELECT *
       FROM games
       WHERE status = '%<status>s'
+        AND title ~ :title
```

Comment:
why are you using a regex

Reply:
This method for creating SQL queries is a bit new to me. I couldn't figure out a way to _not_ use title in the query if none was specifed. Ideally the title parameter wouldn't be there at all when none was specified, then '=' when it is.